8_viz.yml

```diff
@@ -70,3 +71,34 @@ targets:
 
   8_viz/out/lakes_summary_fig.html:
     command: gd_get('8_viz/out/lakes_summary_fig.html.ind')
+
+  ##### Adding summary and visual of data that we are thinking of using for TOHA
+
+  # These datasets help visualize and summarize the data we are using for TOHA models.
+  # They were downloaded from here: https://drive.google.com/drive/u/4/folders/13w3QDXicjHVymVW1NeyJpJqVRZ7KvlTD
+  # Then uploaded to this pipeline GD folder & then incorporated into the pipeline using `gd_confirm_posted`.
+  8_viz/in/WAE_numYears_meanCPUE.csv:
+    command: gd_get(I("8_viz/in/WAE_numYears_meanCPUE.csv.ind"))
```

Comment:
why does it need to be wrapped in `I()` here?

Reply:
That would be copy-paste spaghetti. Hold on a sec